src/lib.rs

```diff
@@ -42,6 +42,30 @@
 use std::collections::HashMap;
 use testcontainers::{core::WaitFor, Container, Image};
 
+#[derive(Clone, Copy, Debug)]
+pub enum Neo4jLabsPlugin {
+    Apoc,
+    ApocCore,
+    Bloom,
+    Streams,
+    GraphDataScience,
+    NeoSemantics,
```

Comment:
If you want to also support arbitrary custom plugins, you can use 

```suggestion
    NeoSemantics,
    Custom(String),
```

Would have to remove the `Copy` from derive though.